projects/tinkerbell/charts/docker/linux/dhcrelay/docker/linux/Dockerfile

```diff
@@ -0,0 +1,15 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-minimal-base-glibc
```

Comment:
the file path/name is odd.

Reply:
this file shouldn't be included in the pr, I'll remove it